enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/mock/MockOnboardingProvider.java

```diff
@@ -48,4 +49,34 @@ public void sendOtpCode(SendOtpCodeRequest request) {
         logger.info("Send OTP code called, user ID: {}, OTP code: {}, resend: {}", userId, otpCode, resend);
     }
 
+    @Override
+    public String fetchConsent(final ConsentTextRequest request) {
+        final UUID processId = request.getProcessId();
+        final String userId = request.getUserId();
+        final String consentType = request.getConsentType();
+        final Locale locale = request.getLocale();
+        logger.info("Fetching consent for processId: {}, userId: {}, consentType: {}, locale: {}", processId, userId, consentType, locale);
+
+        return "<html><body>" +
```

Comment:
This is the main change of this PR, other changes are just merge from upstream.
@kober32 Would you like to have another text for initial testing?